internal/buildengine/testdata/alpha/alpha.go

```diff
@@ -18,7 +18,7 @@ type EchoResponse struct {
 }
 
 //ftl:verb
-func Echo(ctx context.Context, req EchoRequest) (EchoResponse, error) {
-	ftl.Call(ctx, other.Echo, other.EchoRequest{})
+func Echo(ctx context.Context, req EchoRequest, oc other.EchoClient) (EchoResponse, error) {
```

Comment:
How do we know which parameter is the request? Must it always be the 2nd?

Reply:
in this implementation yes, though maybe it could be inferred? the first non-resource-typed param (where verb clients are functions and other resources are known types, e.g. ftl.PostgresDatabase etc.) we encounter

maybe worth landing the other resource types first and then revisiting this

Reply:
current approach allows a request as the 2nd parameter, optionally. if it's not there we just start consuming resources